.gitignore

```diff
@@ -263,3 +263,10 @@ paket-files/
 /src/Db/PKSimDB - Kopie.mdb
 /src/Db/Transfer.mdb
 /src/Db/Transfer.ldb
+/7.2_PKSim.exe.lnk
```

Comment:
@Yuri05 What's all that?

Reply:
@Yuri05 Can you clean that up or is this required?

Reply:
just added some of my local files to the ignore list, so I don't see them every time in the Add-dialog.
Problematic somehow?